.github/workflows/rust.yml

```diff
@@ -105,31 +41,14 @@ jobs:
         PYTHON3: "C:\\mozilla-build\\python3\\python3.exe"
         LIBCLANG_PATH: "C:\\ProgramData\\scoop\\apps\\llvm\\current\\lib"
       run: |
-        cargo test --verbose ${{ matrix.features }} --lib
-  Integrity:
-    runs-on: ubuntu-latest
-    steps:
-    - uses: actions/checkout@v2
-    - name: Get mozjs
-      run: |
-        bash ./mozjs/etc/get_mozjs.sh
-    - name: Apply patch
-      run: |
-        python3 ./mozjs/etc/update.py --no-commit mozjs.tar.xz
-        # Run `git add` here to force CRLF converted into LF
-        # so that we can check diff properly in next run
-        git add --all mozjs
-    - name: Check patch integrity
-      working-directory: ./mozjs
-      # Because we've added files in previous run, we need to
-      # check diff with `--staged`.
-      run: |
-        git diff --staged --no-ext-diff --quiet --exit-code
+        rustup install nightly-2022-11-20
+        rustup default nightly-2022-11-20
+        cargo test --verbose ${{ matrix.features }} --release --lib
```

Comment:
The mozjs binary tests are being skipped on Windows due to the `--lib` option passed to `cargo test`. We can just remove it to test both mozjs & mozjs-sys:

Here are the logs from the [last successful windows job](https://github.com/servo/mozjs/actions/runs/4033746654/jobs/6934466205)
```
    Finished release [optimized] target(s) in 9m 46s
     Running `D:\a\mozjs\mozjs\target\release\deps\mozjs-0a886cf876b6e577.exe`

running 0 tests

test result: ok. 0 passed; 0 failed; 0 ignored; 0 measured; 0 filtered out; finished in 0.00s

     Running `D:\a\mozjs\mozjs\target\release\deps\mozjs_sys-0bd5cea302288402.exe`

running 694 tests
```

Reply:
Woah. Great catch!